### ISSUE TITLE
Remove or reorder columns in guild view compare screen

### ISSUE THREAD

**Issue by tbelhalfaoui** [guildai/guildai#109]
The "Compare runs" screen of Guild View is really handy, but the number of columns can increase rapidly (especially since dependencies are listed as variables).

Therefore, it would be nice if one could specify which columns one wants to display, e.g.:
* right click > hide on the column header,
* or a drop-down checkbox list to select the column names,
* or drag and drop column header to reorder.

This would mimic the behavior of the `guild compare --strict-cols` command.

In all cases, it would be handy to have a horizontal scrollbar, which afaik is currenty missing.

Thanks!

**Comment by tbelhalfaoui** [comment]
Edit:
The horizontal scrollbar is visible, but it is not at the bottom of the screen, as expected.
If there are many experiments to compare (typically, on my screen, more than 11), I have to first scroll all the way down the table to see the horizontal scrollbar, which is somehow coutnerintuitive.

I think fixing the horizontal scrollbar position would already be very helpful :)

**Comment by gar1t** [comment]
I agree this is a major usability issue for View. Marking this as high priority so we don't overlook it in the next release.

In the meantime, the console UI `guild compare` might be a bit more direct to get that info. I understand it's not the same as the web based UI, but the info is there and you might find it faster to work with.
